### DISCUSSION TITLE
Using Input component with forwardref and react hook form controller with zod cause page infinity loading without any error

### DISCUSSION THREAD

**Discussion by dpzkf** [tailwindlabs/headlessui#3650] [Help]
**What packages are you using?**

 "@headlessui/react": "^2.1.10", "@hookform/resolvers": "^3.9.1", "react-hook-form": "^7.53.1", "zod": "^3.23.8"

**What browser are you using?**

For example: Chrome

**Describe your issue**

Using Input component with forwardref and react hook form controller with zod cause page infinity loading without any error

**Comment by RobinMalfait** [comment]
Hey!

Can you share a minimal reproduction repo?
